src/views/profiles/ProfileLayout.tsx

```diff
@@ -133,13 +139,30 @@ const ProfileLayout = ({ children, profileData }: ProfileLayoutProps) => {
                 setShowFollowsModal={setShowFollowsModal}
                 showButton={false}
               />
-              <div className="mt-10 flex justify-center lg:justify-start">
+              <div className="mt-10 flex justify-center gap-4 lg:justify-start">
                 <Link href={'/messages?address=' + publicKey} passHref>
                   <a className=" flex max-w-fit items-center  space-x-2 rounded-full px-4 py-2 text-base shadow-lg shadow-black hover:text-gray-300 ">
                     <span>Message</span>
                     <MessagesIcon className="h-5 w-5" />
                   </a>
                 </Link>
+                <div
+                  className="flex max-w-fit cursor-pointer items-center space-x-2 rounded-full px-4 py-2 text-base shadow-lg shadow-black hover:text-gray-300"
+                  onClick={() => setShowQRModal(true)}
+                >
+                  <span>Scan QR</span>
+                </div>
+                {ReactDom.createPortal(
```

Comment:
Neat! Didnt know about this. I wonder if this should be a part of `Modal` though, given that modals always (?) appear on top of everything else.

Reply:
Yeah, we should move that into the Modal with storybook! :) Or potentially sooner